docs/docs/configuration/tooltip.md

```diff
@@ -178,17 +178,23 @@ The tooltip items passed to the tooltip callbacks implement the following interf
 
```

Comment:
May want to rename this section to "Tooltip Item Context"

Reply:
done